packages/terra-disclosure-manager/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
```

Comment:
Call this out as a breaking change
https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/CHANGELOG.md#breaking-changes---upgrade-guide
Also, we should add an upgrade guide.

```suggestion
### Breaking Change
```